AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift.xcodeproj/project.pbxproj

```diff
@@ -402,6 +430,65 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
```

Comment:
Since the demo app is meant to be used by external developers, we try to leave project in its original state where possible, which means omitting checking of CocoaPods updates to `.xcodeproj` (since the `Podfile` developers use and we use in our workspace is different).